Formula/demumble.rb

```diff
@@ -0,0 +1,22 @@
+class Demumble < Formula
+  desc "More powerful symbol demangler (a la c++filt)"
+  homepage "https://github.com/nico/demumble"
+  url "https://github.com/nico/demumble/archive/refs/tags/v1.2.2.tar.gz"
+  sha256 "13f89330abbae24262d1c7a0717c25e71737b727e72db78ff77d8ca35566c22f"
```

Comment:
You have a checksum mismatch:
```
Error: SHA256 mismatch
Expected: 13f89330abbae24262d1c7a0717c25e71737b727e72db78ff77d8ca35566c22f
  Actual: 663e5d205c83cc36a257bb168d3ecbc2a49693088c0451b2405d25646651c63e
```
You probably got this from piping `curl` to `sha256sum`, or similar:
```
❯ curl -s https://github.com/nico/demumble/archive/refs/tags/v1.2.2.tar.gz | sha256sum
13f89330abbae24262d1c7a0717c25e71737b727e72db78ff77d8ca35566c22f  -
```
But this just gives you the checksum of GitHub's redirection stub:
```
❯ curl -s https://github.com/nico/demumble/archive/refs/tags/v1.2.2.tar.gz
<html><body>You are being <a href="https://codeload.github.com/nico/demumble/tar.gz/refs/tags/v1.2.2">redirected</a>.</body></html>
```
Instead, what you want is
```
❯ curl -Ls https://github.com/nico/demumble/archive/refs/tags/v1.2.2.tar.gz | sha256sum
663e5d205c83cc36a257bb168d3ecbc2a49693088c0451b2405d25646651c63e  -
```

Reply:
Guilty as charged... And I thought I watched for the correct size download.